easybuild/easyconfigs/b/BWA/BWA-0.7.17-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+#
+# Version >= 0.7.15
+# Author: Adam Huffman
+# The Francis Crick Institute
+#
+# Note that upstream development is mainly at: https://github.com/lh3/bwa
+##
+
+name = 'BWA'
+version = '0.7.17'
+
+homepage = 'http://bio-bwa.sourceforge.net/'
+
+description = """
+ Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively
+ short nucleotide sequences against a long reference sequence such as the human
+ genome.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://github.com/lh3/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['980b9591b61c60042c4a39b9e31ccaad8d17ff179d44d347997825da3fdf47fd']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+dependencies = [('zlib', '1.2.11')]
```

Comment:
@ItIsI-Orient Any particular reason why `Perl` is no longer listed as a dependency? It should be, because the BWA installation also includes some Perl scripts...